tests/libvmi/vmi.go

```diff
@@ -71,6 +71,13 @@ func WithResourceMemory(value string) Option {
 	}
 }
 
+// WithNamespace specifies the vmi namespace.
+func WithNamespace(name string) Option {
```

Comment:
Please refer to the comment regarding the namespace in `newRandomCirrosVMIWithIstioSidecarAndDefaultServiceAccountAndNetworkAndInterface`.